atcoder-problems-frontend/src/pages/Internal/VirtualContest/ShowContest/FirstAcceptedRow.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { ProblemId, UserId } from "../../../../interfaces/Status";
+import { ReducedProblemResult } from "./ResultCalcUtil";
+import { FirstAccepted, FirstAcceptedCell } from "./FirstAcceptedCell";
+
+interface FirstAcceptedRowProps {
+  start: number;
+  userIds: UserId[];
+  problemIds: string[];
+  resultsByUser: Map<UserId, Map<ProblemId, ReducedProblemResult>>;
+  showRating: boolean;
+}
+
+export const FirstAcceptedRow: React.FC<FirstAcceptedRowProps> = (props) => {
+  const { start, userIds, problemIds, resultsByUser, showRating } = props;
+
+  const fastestByProblem = new Map<string, FirstAccepted>();
```

Comment:
```suggestion
  const fastestByProblem = new Map<ProblemId, FirstAccepted>();
```
気休めですが、せっかくなので何のstringか明示しておきましょう。